web/src/app/schedule-view/component.ts

```diff
@@ -167,6 +167,24 @@ export class ScheduleViewComponent implements OnInit, OnDestroy, AfterViewInit {
         });
   }
 
+  public copyScheduleLink () : void {
```

Comment:
It looks like the preferred method for copying is this: https://gist.github.com/sandcastle/00aaa8a820061c899edf76c3ed3c8bac

Also, we should move the copy to clipboard function out to a service so it can be re-used if needed, and just to the url building in the controller.

Alternatively, we could use a small library like [this](https://www.npmjs.com/package/ts-clipboard)